src/fparser/two/tests/fortran2008/test_end_submodule_stmt_r1119.py

```diff
@@ -41,21 +41,21 @@
 from fparser.two.Fortran2003 import NoMatchError
 from fparser.two.Fortran2008 import End_Submodule_Stmt
 from fparser.two.parser import ParserFactory
-# this is required to setup the fortran2008 classes
-_ = ParserFactory().create(std="f2008")
 
 
 def test_simple_1():
-    '''Test the parsing of a minimal end-submodule statement'''
+    '''Test the parsing of a minimal end-submodule statement.'''
+    _ = ParserFactory().create(std="f2008")
```

Comment:
It would be better to use a set-up fixture in this test file so that you only have one place where you create the f2008 parser.

Reply:
Done. Created a fixture in a conftest.py file and use it for all tests that previously explicitly used ParserFactory.